### ISSUE TITLE
Make DartPad example button accessible using keyboard

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10544]
From https://buganizer.corp.google.com/issues/337848673

### Observed Results

Navigating through the interactive code example, noticed that the buttons inside are not reachable using keyboard controls.

* [Screencast VoiceOver](http://screencast/cast/NjcxOTU4MjYyMTQwMTA4OHxkZGRhYWVhNi0xYQ)
* [Screencast ChromeVox](http://screencast/cast/NjQzMzgyMjkwMzEwNzU4NHw5Zjc0ZjU0ZC0wOQ)

### Expected Results

Ensure that the buttons inside the interactive code should be managed by keyboard controls.

### User Impact

Screen reader and Cognitive users are confused because the buttons are announced and sometimes no but they cannot be managed by key controls.

### Steps to Reproduce

1. Enabled VoiceOver using <kbd>Command</kbd> + <kbd>F5</kbd> also ChromeVox using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Z</kbd>
1. Open the following page: https://docs.flutter.dev/cookbook/design/snackbars#interactive-example
1. Continue the linear navigation using the <kbd>Tab</kbd> and <kbd>Arrow</kbd> keys and reach the interactive example code.
1. Observe the results.

### GAR Assessment

[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR Details

[1.2. Can all controls be reached and used via the keyboard without using a mouse?](go/GAR-web#keyboard)

### Environment Details

Machine: MAC/ChromeBook
Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
Screen reader: VoiceOver / ChromeVox.
Bug filling time: 15 mins.

**Comment by parlough** [comment]
Closing this in favor of https://github.com/dart-lang/dart-pad/issues/2488 which tracks DartPad accessibility work. But in this specific case, I can `Esc+Tab` to navigate to the example's rendered button. However the ability to do so isn't clear, so I opened https://github.com/dart-lang/dart-pad/issues/3197 to track surfacing this ability.
